winrm/protocol.py

```diff
@@ -395,6 +395,31 @@ def cleanup_command(self, shell_id, command_id):
         # TODO change assert into user-friendly exception
         assert uuid.UUID(relates_to.replace('uuid:', '')) == message_id
 
+    def send_command_input(self, shell_id, command_id, stdin_input):
+        """
+        Send input to the given shell and command.
+        @param string shell_id: The shell id on the remote machine.
+         See #open_shell
+        @param string command_id: The command id on the remote machine.
+         See #run_command
+        @param string stdin_input: The input unicode string to be sent.
+        @return: None
```

Comment:
My mistake.  I was returning because there might be additional checks that need to be performed to ensure that there are no exceptions to be raised. Although, at the current moment, I do no know what they would be and haven't seen any errors till this point in time.
code has been adjusted.